client/client.go

```diff
@@ -136,6 +136,13 @@ func (c *Client) useInsecureHTTPClient(insecure bool) *http.Transport {
 }
 
 func (c *Client) MakeRestRequest(method string, path string, body *container.Container, authenticated bool) (*http.Request, error) {
+	if strings.HasPrefix(path, "/") {
+		path = path[1:]
+	}
+	split := strings.Split(path, "/")
+	if c.platform == "nd" && split[1] == "v2" {
```

Comment:
when platform is nd, you can add /mso/ to any path. Did you had an issue with some?

Reply:
when we do authentication and platform is nd, we still need to use api V1